charts/v2.0.0-alpha.1/azuredisk-csi-driver/templates/crd-csi-snapshot-ga.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/419"
   creationTimestamp: null
   name: volumesnapshots.snapshot.storage.k8s.io
+  labels: 
```

Comment:
I don't believe we should include Azure Disk CSI Driver specific labels for the CSI CRDs. These could be installed by other drivers (e.g. Azure Files CSI Driver) and should be the same.